deps/wazuh_testing/wazuh_testing/scripts/simulate_agents.py

```diff
@@ -108,7 +108,10 @@ def create_agent(args, custom_labels):
         'syscollector_frequency': args.syscollector_frequency,
         'syscollector_event_types': args.syscollector_event_types,
         'syscollector_legacy_messages': args.syscollector_legacy_messages,
-        'syscollector_packages_vuln_content': args.syscollector_packages_list_file
+        'syscollector_packages_vuln_content': args.syscollector_packages_list_file,
+        'vulnerability_legacy_messages': args.vulnerability_legacy_messages,
```

Comment:
We should not support legacy messages for the vulnerability simulator. Those are not supported by new Vulnerability Detection module
```suggestion
```